app/controllers/api/v1/trips_controller.rb

```diff
@@ -25,4 +26,16 @@ def create
    def trip_params
       params.require(:trip).permit(:name, :location, :start_date, :end_date, :status, :total_budget, :user_id)
    end
+
+   def set_trip
+      @trip = Trip.find(params[:id])
+   end
+
+   def confirm_user
+      render_user_error(params) if @trip.user_id != params[:user_id].to_i 
+   end
+
+   def filter_user_trips
+      @trips = Trip.where(user_id: params[:user_id])
+   end
```

Comment:
Should this be a model method?

Reply:
Since it is using .where

Reply:
I think something simple like this is not necessary. Only if it was a more elaborated query, but I can def put this in the model if you think it's more appropriate.

Reply:
Just created a model method